humans/Object item
float64       confidence